source/guide_minio.rst

```diff
@@ -0,0 +1,152 @@
+.. author:: Marius Bertram <marius@brtrm.de>
+
+.. tag:: file-storage
+.. tag:: lang-go
+
+.. highlight:: console
+
+.. sidebar:: MinIO
+
+  .. image:: _static/images/minio_logo.png
+      :align: center
+
+#####
+MinIO
+#####
+
+.. tag_list::
+
+MinIO_ is an open source object storage server,
+compatible with Amazon S3 writen in go.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Web Backends <web-backends>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+MinIO is licensed under Apache-2.0
+
+     https://github.com/minio/minio#license
+
+
+Prerequisites
+=============
+Download the latest MinIO binary into your bin folder and make it executable:
+
+.. code-block:: console
+
+    [isabell@stardust ~]$ wget https://dl.min.io/server/minio/release/linux-amd64/minio -O /home/isabell/bin/minio
+    [isabell@stardust ~]$ chmod +x /home/isabell/bin/minio
+    [isabell@stardust ~]$
+
+Create a folder for your data:
+
+.. code-block:: console
+
+    [isabell@stardust ~]$ mkdir /home/isabell/minio
```

Comment:
Just use `mkdir minio` here or `mkdir ~/minio` here. All guides assume that the sessions starts out in the home dir.

Reply:
Oh, minio auto-creates the directory. So we can skip this step.